functions/process-xml-to-json/1.0/function.json

```diff
@@ -0,0 +1,32 @@
+{
+  "description": "Description",
```

Comment:
Can you add a description

Reply:
See _chore: update naming and description_